Initial commit for Italian texture localization

Macapple96 opened the pull request:
Contributing italian texture localization to the project!

- Added `it` locale;
- Added `it/pdn/` folder which contains paintdotnet source image files for easier editing;
-- Includes manually "cleaned up" blank jokers as layers, separated from text; 
- Added overrides for italian texture generation in `consumables.json`;
- Generated italian spritesheets in `dist/it/`;

No mod uses these files yet; Playtested with manual file substitution method.
Due to italian grammar, some decorative elements in booster packs (such as _Jumbo Arcana packs_) may look a bit off.

Macapple96 committed:
Initial commit for Italian texture localization

Signez commented:
So cool! I'll take a few days to look into it (especially the Paint.net files, I'll have to boot up my Windows PC to check them out), but it's very cool to see another contribution to this pipeline with another language! Thanks a lot.

Signez commented on locales/it/jokers/j_todo_list.png:
I am afraid it's impossible to read “JOKER” by reading each of the first letters of the todolist in the Italian translation (it reads "JPKER"), whereas it's possible in every other translation we already merged and in the original asset.

|Original|French|Romanian|Italian|
|--|--|--|--|
|![Original](https://github.com/Signez/balatro-sprites-i18n/assets/411336/c427e74f-448c-4b8a-8934-67f614f788be)|![French](https://github.com/Signez/balatro-sprites-i18n/assets/411336/2425b0a2-9eb5-4d0c-a26e-8cd532d191b5)|![Romanian](https://github.com/Signez/balatro-sprites-i18n/assets/411336/22827f9c-7af0-43cb-8209-45bca526d91b)|![Italian](https://github.com/Signez/balatro-sprites-i18n/assets/411336/7048445c-fe0e-4629-bfc1-0afae8af284e)|

If you feel it's too hard to fix / you don't care, it's fine, feel free to tell me and I can merge it as-is.

Macapple96 replied:
Hello! I just updated the sprite as requested. 

I'm a bit embarassed to admit the reason it didn't read 'JOKER' from the beginning was that I _never_ noticed in the first place 😆 

Here's the new version!

| |todo_list.png|reserved_parking.png|
| :--- | :---: | :---: |
| *New* | ![j_todo_list](https://github.com/Signez/balatro-sprites-i18n/assets/1537826/d7f786a1-3585-4b1f-8269-0ce2018d7a42) | ![j_reserved_parking](https://github.com/Signez/balatro-sprites-i18n/assets/1537826/b62ebdd9-f245-4b67-8133-e0e442df3775)
| *Old* | ![tmpF7B6 tmp](https://github.com/Signez/balatro-sprites-i18n/assets/1537826/aa1ff6fd-001a-4c9c-99c1-e922a6ce07ff) | ![tmp2464 tmp](https://github.com/Signez/balatro-sprites-i18n/assets/1537826/8c515a7d-c8ff-4f87-adf9-629e2e7f7b84) |

Signez reviewed (changes requested):
Great work! I mainly have one remark that could improve that translation before a merge.

Macapple96 committed:
Conform pull request changes
- Edited `j_todo_list.png` to read 'JOKER'
- Edited `j_reserved_parking.png` with taller letters to match original sprite better
- Updated `.pdn` source files for both jokers

Signez commented:
Thanks again for your contributions 🎉
